tests/src/common/TestUtils.java

```diff
@@ -326,4 +332,34 @@ public static RunResult runCmd(int expectedExitCode, int timeoutMilli, File dir,
         rr.validateExitCode(expectedExitCode);
         return rr;
     }
+
```

Comment:
i'll add that comment.  i wanted to keep these debugged utility methods hanging around, just in case they are useful at some later point.